### ISSUE TITLE
Ensure councillor data is current

### ISSUE THREAD

**Issue by henare** [openaustralia/planningalerts#1013]
It's really important that we keep councillor data up to date or Ask Your Councillors becomes something that doesn't work for anyone. Before we move off the project, what are we planning to do to ensure we keep on top of this?

**Comment by equivalentideas** [comment]
One important step to this to load in the new councillors who have been elected since we launched Ask Your Councillors.

We also want to keep existing councilors around so their replies are visible, but not have people write to them. I'm going to split that aspect into a separate issue.

**Comment by equivalentideas** [comment]
We're currently blocked from loading NSW councillors by this bug in WriteIt https://github.com/ciudadanointeligente/write-it/issues/983

**Comment by equivalentideas** [comment]
I've added a way to mark councillors as 'current' or not ( #1074 ). I've updated the data for melbourne city council post Oct 2016 elections and loaded them into production, which also disabled writing to the councillors who are gone.

The next step is update the councillors data, and improve that process.

As part of this issue we could also do a callout for volunteers to help with it. Feels a bit tricky to do at the moment.
